.env-sample

```diff
@@ -1,6 +1,16 @@
+# set up database
```

Comment:
Could we append all Database credentials with DB_ for uniformity and to avoid naming conflicts e.g. DB_USER instead of USER. You could also tag all the variables with SCORPION_ to avoid any conflicts with other existing project's environment variables. e.g SCORPION_DB_USER